projects/ad9081_fmca_ebz_x_band/zcu102/system_bd.tcl

```diff
@@ -166,8 +168,48 @@ create_bd_port -dir O tdd_tx_stingray_en
 
 set tdd_sync_in_net [get_bd_nets -of_objects [find_bd_objs -relation connected_to [get_bd_pins axi_tdd_0/sync_in]]]
 set tdd_sync_in_pin [get_bd_pins axi_tdd_0/sync_in]
-ad_disconnect $tdd_sync_in_net $tdd_sync_in_pin
+
+set adc_do_m_axis_clk_net [get_bd_nets -of_objects [find_bd_objs -relation connected_to [get_bd_pins $adc_data_offload_name/m_axis_aclk]]]
+set adc_do_m_axis_clk_pin [get_bd_pins $adc_data_offload_name/m_axis_aclk]
+
+set adc_dma_s_axis_clk_net [get_bd_nets -of_objects [find_bd_objs -relation connected_to [get_bd_pins axi_mxfe_rx_dma/s_axis_aclk]]]
+set adc_dma_s_axis_clk_pin [get_bd_pins axi_mxfe_rx_dma/s_axis_aclk]
+
+set adc_do_m_axis_rst_net [get_bd_nets -of_objects [find_bd_objs -relation connected_to [get_bd_pins $adc_data_offload_name/m_axis_aresetn]]]
+set adc_do_m_axis_rst_pin [get_bd_pins $adc_data_offload_name/m_axis_aresetn]
+
+set adc_dma_m_dest_rst_net [get_bd_nets -of_objects [find_bd_objs -relation connected_to [get_bd_pins axi_mxfe_rx_dma/m_dest_axi_aresetn]]]
+set adc_dma_m_dest_rst_pin [get_bd_pins axi_mxfe_rx_dma/m_dest_axi_aresetn]
+
+set hp1_fdp_aclk_net [get_bd_nets -of_objects [find_bd_objs -relation connected_to [get_bd_pins sys_ps8/saxihp1_fpd_aclk]]]
+set hp1_fdp_aclk_pin [get_bd_pins sys_ps8/saxihp1_fpd_aclk]
+
+set axi_hp1_interconnect_aclk_net [get_bd_nets -of_objects [find_bd_objs -relation connected_to [get_bd_pins axi_hp1_interconnect/aclk]]]
+set axi_hp1_interconnect_aclk_pin [get_bd_pins axi_hp1_interconnect/aclk]
+
+ad_disconnect $hp1_fdp_aclk_net              $hp1_fdp_aclk_pin
+ad_disconnect $tdd_sync_in_net               $tdd_sync_in_pin
+ad_disconnect $adc_do_m_axis_clk_net         $adc_do_m_axis_clk_pin
+ad_disconnect $adc_dma_s_axis_clk_net        $adc_dma_s_axis_clk_pin
+ad_disconnect $adc_do_m_axis_rst_net         $adc_do_m_axis_rst_pin
+ad_disconnect $adc_dma_m_dest_rst_net        $adc_dma_m_dest_rst_pin
```

Comment:
Since _m_dest_axi_aclk_ is not changed, then the reset _m_dest_axi_aresetn_ should be left connected to its original source.